Templates for spider

NohTow opened the pull request:
Templates for the spider dataset.
One major issue was the variation in the writing style of question (some are not really "questions"), so I tried to make it sounds like exercise statements, hope it is good enough.

srush commented on templates/spider/templates.yaml:
```diff
@@ -0,0 +1,28 @@
+templates:
+  4f63f872-06e2-4503-a440-c1908d5cb642: !Template
+    id: 4f63f872-06e2-4503-a440-c1908d5cb642
+    jinja: Can you give me the SQL request which respond to "{{question}}" ? ||| {{query}}
```
Respond should be "corresponds"

srush commented on templates/spider/templates.yaml:
```diff
@@ -0,0 +1,28 @@
+    reference: ''
+  ba6415e9-9467-4519-8789-57d65524464e: !Template
+    id: ba6415e9-9467-4519-8789-57d65524464e
+    jinja: The SQL query associated to the question "{{question}}" is ||| {{query}}
```
"associated with"

srush commented on templates/spider/templates.yaml:
```diff
@@ -0,0 +1,28 @@
+    reference: ''
+  e843ba1e-09af-48ba-92ca-1b6032404985: !Template
+    id: e843ba1e-09af-48ba-92ca-1b6032404985
+    jinja: 'To respond to "{{question}}", I can use the SQL query: ||| {{query}}'
```
Instead of "respond to" , I would say "lookup "

srush reviewed (changes requested):
These are great. But can you also include the reverse entries. "I would describe the SQL statement ... to a new learner as ||| ..."

NohTow commented:
I added your corrections/recommendations and checked the main task box.
I also added the reverse entries, feel free to tell me if something is odd.
